### PR TITLE
Fixes Malware Detection

### PR DESCRIPTION
**Pull request by Varunshiyam**
# Related Issues or bug

This project demonstrates the use of deep learning for malware detection on Android devices. The dataset, containing behavioral characteristics of both benign and malware apps, is analyzed and preprocessed. Feature selection is performed to identify the most relevant attributes. A deep neural network is then constructed and trained using an Adam optimizer and sparse categorical cross-entropy loss. The model's performance is evaluated and visualized, showing high accuracy in classifying malware. Further training with an SGD optimizer is applied to potentially enhance the model's performance. This project highlights the effectiveness of deep learning in identifying malicious software based on behavioral patterns.

Fixes: #701 

# Proposed Changes
Added up a Project File in the format of .ipynb with detailed Comments for easy Understanding

### CODE REVIEW

**Comment by UppuluriKalyani** [comment]
@Varunshiyam add proper readme file

**Review by SaiNivedh26 — CHANGES_REQUESTED** [review]
Add a proper .readme file

**Comment by Varunshiyam** [comment]
@SaiNivedh26 @UppuluriKalyani  I had added up a Readme File , Kindly review it.

**Review by UppuluriKalyani — APPROVED** [review]

**Comment by UppuluriKalyani** [comment]
@Varunshiyam why did you deleted the file?
